qiita_pet/handlers/api_proxy/prep_template.py

```diff
@@ -141,18 +141,13 @@ def prep_template_ajax_get_req(user_id, prep_id):
     investigation_type = pt.investigation_type
 
     download_prep_id = None
-    download_qiime_id = None
```

Comment:
Realized that it will be better to stop displaying the qiime file to avoid confusion.